### ISSUE TITLE
./setup-hayagriva doesn't work

### ISSUE THREAD

**Issue by jcbhmr** [jcbhmr/setup-hayagriva#2]
### Discussed in https://github.com/orgs/typst-community/discussions/7

<div type='discussions-op-text'>

<sup>Originally posted by **agerlach** February  6, 2024</sup>
I am having troubles getting the setup-hayagriva GH action to work. I'm not quite sure what the issue is. Any guidance is appreciated. FYI, I am relatively new to GH actions. I essentially forked this repo https://github.com/mintyfrankie/brilliant-CV and extended the workflow to use `setup-hayagriva`.

I am using the following in `.github/workflows/compile.yml`

```
name: Check PDF Compilation

on:
  push: {}
  pull_request: {}

jobs:
  build:
    name: compile-check
    runs-on: ubuntu-latest

    steps:
      - name: Checkout
        uses: actions/checkout@v4
        with:
          submodules: 'recursive'

      - name: Setup Typst
        uses: typst-community/setup-typst@v3

      - name: Setup Hayagriva
        uses: typst-community/setup-hayagriva@v1.0.1
            

      - run: hayagriva ./src/bibliography.bib |> ./src/bibliography.yml
      - run: typst compile cv.typ --font-path ./src/fonts
      - run: typst compile letter.typ --font-path ./src/fonts

      - name: Upload Artifacts
        uses: actions/upload-artifact@v3
        with:
          name: output
          path: ./*.pdf
          if-no-files-found: error

```

But I am getting the following error during the `Setup Hayagriva` stage.

```
Run typst-community/setup-hayagriva@v1.0.1
Run cargo-bins/cargo-binstall@main
Run curl -L --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/cargo-bins/cargo-binstall/main/install-from-binstall-release.sh | bash
++ mktemp -d
+ cd /tmp/tmp.8zZ4YtDqmu
+ base_url=https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-
++ uname -s
+ os=Linux
+ '[' Linux == Darwin ']'
+ '[' Linux == Linux ']'
++ uname -m
+ machine=x8[6](https://github.com/agerlach/brilliant-CV/actions/runs/7801271579/job/21275970222#step:3:7)_64
+ target=x86_64-unknown-linux-musl
+ '[' x86_64 == armv[7](https://github.com/agerlach/brilliant-CV/actions/runs/7801271579/job/21275970222#step:3:9) ']'
+ url=https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-x86_64-unknown-linux-musl.tgz
+ tar -xvzf -
+ curl -L --proto =https --tlsv1.2 -sSf https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-x86_64-unknown-linux-musl.tgz
cargo-binstall
+ ./cargo-binstall -y --force cargo-binstall
 WARN Failed to retrieve token from `gh auth token` err=Custom { kind: Other, error: "process exited with `exit status: 1`" }
 WARN Failed to read git credential file
 INFO resolve: Resolving package: 'cargo-binstall'
 INFO resolve: Verified signature for package 'cargo-binstall': gh=cargo-bins/cargo-binstall git=3f271cce2b581890bb2[11](https://github.com/agerlach/brilliant-CV/actions/runs/7801271579/job/21275970222#step:3:14)a740bfde8073f[15](https://github.com/agerlach/brilliant-CV/actions/runs/7801271579/job/21275970222#step:3:18)0150 ts=2024-02-04T02:[16](https://github.com/agerlach/brilliant-CV/actions/runs/7801271579/job/21275970222#step:3:19):15.660Z run=7770957747
 WARN The package cargo-binstall v1.6.2 (x86_64-unknown-linux-gnu) has been downloaded from github.com
 INFO This will install the following binaries:
 INFO   - cargo-binstall (cargo-binstall -> /home/runner/.cargo/bin/cargo-binstall)
 INFO Installing binaries...
 INFO Done in 1.8[25](https://github.com/agerlach/brilliant-CV/actions/runs/7801271579/job/21275970222#step:3:28)876665s
+ CARGO_HOME=/home/runner/.cargo
+ [[ :/snap/bin:/home/runner/.local/bin:/opt/pipx_bin:/home/runner/.cargo/bin:/home/runner/.config/composer/vendor/bin:/usr/local/.ghcup/bin:/home/runner/.dotnet/tools:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin: == *\:\/\h\o\m\e\/\r\u\n\n\e\r\/\.\c\a\r\g\o\/\b\i\n\:* ]]
Error: Can't find 'action.yml', 'action.yaml' or 'Dockerfile' under '/home/runner/work/brilliant-CV/brilliant-CV/setup-hayagriva'. Did you forget to run actions/checkout before running your local action?
```

</div>

cc original poster @agerlach

**Comment by jcbhmr** [comment]
provided details in https://github.com/orgs/typst-community/discussions/7. intended solution is to avoid ./setup-hayagriva in action.yml root

**Comment by jcbhmr** [comment]
Fix should be live in the v1 tag and the v1.0.2 tag 😊

**Comment by agerlach** [comment]
@jcbhmr Works like a charm. Thanks
